### PR TITLE
Experimental test to see if we can set creation date

### PR DESCRIPTION
**Pull request by sbryzak**
Signed-off-by: sbryzak <sbryzak@gmail.com>

### CODE REVIEW

File: test/e2e/usersignup_test.go [hunk 1]
```diff
@@ -295,6 +296,22 @@ func (s *userSignupIntegrationTest) TestTransformUsername() {
 	}
 }
 
+func (s *userSignupIntegrationTest) TestUserSignupModifiedCreationDate() {
+	userSignup := NewUserSignup(s.T(), s.hostAwait, "tango", "tango@xray.com")
+	userSignup.Spec.Approved = true
+	past := time.Now().AddDate(-1, 0, 0)
+	userSignup.ObjectMeta.CreationTimestamp = v1.Time{Time: past}
+
+	err := s.hostAwait.FrameworkClient.Create(context.TODO(), userSignup, CleanupOptions(s.ctx))
```
Comment (alexeykazakov) [inline test/e2e/usersignup_test.go:305]: I guess Create will generate a new timestamp. Try to update the existing resource.

Comment (sbryzak) [reply test/e2e/usersignup_test.go:305]: My guess is the same :)  I'll wait for this test to fail first then I'll try the update existing resource method.

**Pull request closed without merging**